kasa/smart/smartdevice.py

```diff
@@ -512,22 +512,18 @@ def _net_for_scan_info(res):
                 bssid=res["bssid"],
             )
 
-        async def _query_networks(networks=None, start_index=0):
-            _LOGGER.debug("Querying networks using start_index=%s", start_index)
+        async def _query_networks(networks=None):
```

Comment:
There'd be no need to keep this as a nested function now that it's not called recursively.